library/yourls

```diff
@@ -1,20 +1,20 @@
-# this file is generated via https://github.com/YOURLS/docker-yourls/blob/eded14078d93e7c85b9be1cdc5f522f2d669019c/generate-stackbrew-library.sh
+# this file is generated via https://github.com/YOURLS/docker-yourls/blob/a358b4f51a61732a89347ba35d59c7bc4c65f6e6/bin/generate-stackbrew-library.sh
 
 Maintainers: YOURLS <yourls@yourls.org> (@YOURLS),
              Léo Colombaro <git@colombaro.fr> (@LeoColomb)
 GitRepo: https://github.com/YOURLS/docker-yourls.git
 
```

Comment:
You'll want to also include `GitFetch:` in here:

```suggestion
GitRepo: https://github.com/YOURLS/docker-yourls.git
GitFetch: refs/heads/main

```

Reply:
(Sorry, updated my suggestion to use `refs/heads/dist` appropriately.)

It's also kind of strange to have `dist` so divorced from `main`, since if users browse around the repository from links on https://hub.docker.com/_/yourls, they'll not have any of the other bits from the repository.

Reply:
Ah, thanks for the suggestion!

Well, this is my point of view on this divorce:
* Each time a new external contribution is made, the contributor does not understand the way variant images have to be updated. No blame on them, because have distribution files and source templates in the same branch is 
* This makes local maintenance much more straightforward: I don't need a complete bash each time I what to fix a typo.
* `dist` branch does not need to host all the file. I use the assumption if someone want to see any of the other bits, they'll need to go on repo home.
* And last but not least: when accessing dist Dockerfiles from Hub page, the page loaded is the exact commit page and the not the head. So the way for other bits differs very little.

Reply:
For example, this is a very usual contribution: https://github.com/YOURLS/docker-yourls/pull/68/files
Only the template and one of the 3 variant has been updates.
Then:
* Not sure which one the contributor used as a reference
* The CI is less relevant, only 1 variant is fully tested under the PR change
* We need to ask the contributor to run the `update.sh` script or do it ourselves.